.github/workflows/build-images-ci.yaml

```diff
@@ -83,39 +74,11 @@ jobs:
           persist-credentials: false
           ref: ${{ steps.tag.outputs.tag }}
 
-      # Load Golang cache build from GitHub
-      - name: Load ${{ matrix.name }} Golang cache build from GitHub
-        uses: actions/cache@937d24475381cd9c75ae6db12cb4e79714b926ed
-        id: cache
-        with:
-          path: /tmp/.cache/${{ matrix.name }}
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}-${{ matrix.name }}-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}-${{ matrix.name }}-
-            ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}-
-            ${{ runner.os }}-go-
-
-      - name: Create ${{ matrix.name }} cache directory
-        if: ${{ steps.cache.outputs.cache-hit != 'true' }}
-        shell: bash
-        run: |
-          mkdir -p /tmp/.cache/${{ matrix.name }}
-
-      # Import GitHub's cache build to docker cache
-      - name: Copy ${{ matrix.name }} Golang cache to docker cache
-        uses: docker/build-push-action@a66e35b9cbcf4ad0ea91ffcaf7bbad63ad9e0229
-        with:
-          context: /tmp/.cache/${{ matrix.name }}
-          file: ./images/cache/Dockerfile
-          push: false
-          platforms: linux/amd64
-          target: import-cache
```

Comment:
Why was this removed?

Reply:
We don't use build caches on older branches, that's why it was removed.